tests/Functional/AutoincrementColumnTest.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+namespace Doctrine\DBAL\Tests\Functional;
+
+use Doctrine\DBAL\Schema\Table;
+use Doctrine\DBAL\Tests\FunctionalTestCase;
+
+use function strtolower;
+
+class AutoincrementColumnTest extends FunctionalTestCase
+{
+    /** @var bool */
+    private static $tableCreated = false;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        if (! self::$tableCreated) {
+            $table = new Table('auto_increment_table');
+            $table->addColumn('id', 'integer', ['autoincrement' => true]);
+            $table->addColumn('test_int', 'integer');
```

Comment:
Why is this column needed? The `id` column should be sufficient to test how autoincrement works.

Reply:
SQLite fails on `insert('autoincrement_table', [])` which would be the way to go here if we were just having a single autoincrement column in the table. Will change anyway so we do not get confused.

Reply:
Please try using `$platform->etEmptyIdentityInsertSQL()` instead:
https://github.com/doctrine/dbal/blob/891ec185b0a0740a0dfeede0a3eddd2f399bb009/src/Platforms/AbstractPlatform.php#L3421-L3424

Reply:
Done, thanks